tests/autoblocks/util.py

```diff
@@ -9,3 +9,7 @@ def make_expected_body(data: Dict) -> bytes:
     https://github.com/Colin-b/pytest_httpx#matching-on-http-body-1
     """
     return json.dumps(data).encode()
+
+
+def decode_request_body(req) -> Dict:
```

Comment:
nit: could just using built in `dict` since you aren't specifying generic values in the return type